res/css/views/right_panel/_UserInfo.scss

```diff
@@ -276,17 +276,17 @@ limitations under the License.
             margin-bottom: 11px;
         }
     }
-}
 
-.mx_UserInfo.mx_BaseCard.mx_UserInfo_smallAvatar {
```

Comment:
Because `.mx_UserInfo.mx_BaseCard` and this line have been on the same level, it took time for me to figure out what prevented transition effect defined below.